Libraries/Text/RCTTextAttributes.m

```diff
@@ -31,6 +31,9 @@ - (instancetype)init
     _textShadowRadius = NAN;
     _opacity = NAN;
     _textTransform = RCTTextTransformUndefined;
+#if TARGET_OS_OSX // TODO(macOS ISS#2323203)
+    _foregroundColor = [NSColor labelColor];
+#endif // TODO(macOS ISS#2323203)
```

Comment:
I did it here because then it’s applied to both text fields and text views, but unsure if this is the right place (perhaps it’s used for other purposes too?). Will look into that more myself as well.

Reply:
This does seem like the appropriate place, as it powers all (but only) text UI elements.